lib/utils/use-ghost-user.js

```diff
@@ -16,6 +16,7 @@ module.exports = function useGhostUser(contentDir) {
         ghostuid = execa.shellSync('id -u ghost').stdout;
         ghostgid = execa.shellSync('id -g ghost').stdout;
     } catch (e) {
+        // CASE: the ghost user doesn't exist, hence can't be used
         if (!e.message.match(/no such user/i)) {
             throw new errors.ProcessError(e);
```

Comment:
@acburdine I am actually not 100% sure if we should throw an error here at all. I can only see two reasons when the command wouldn't work: 
1. Command is not known by Linux
2. User doesn't exist

The first case is not very likely to happen, I assume. And the second case would exactly be the result we want. `false` if the user doesn't exist. 

I will change it for now...

Reply:
That's a fair point.